ui/components/app/confirm-page-container/confirm-page-container-header/index.scss

```diff
@@ -48,4 +48,17 @@
 
     margin-left: 6px;
   }
+
+  .mm-picker-network-div {
+    cursor: default;
+
+    &:active {
+      background-color: initial;
+      color: initial;
+    }
+  }
+}
+
+.confirm-page-container-header-avatar-network-base {
+  padding-right: 1.5px;
 }
```

Comment:
Let's remove these styles :)

Reply:
The letter inside the circle isn't centered, so that's why I added this. The PickerNetwork component does however allow for 'paddingRight' and I tried adding it there. The issue is that paddingRight only accepts a SizeNumber which only allows 4px increments. 

Here's a photo of it without the 1.5px styling.
![Screenshot from 2023-09-20 22-18-47](https://github.com/MetaMask/metamask-extension/assets/51105802/cddb1fe8-ecf5-4f7f-a549-18bc4cb0dd20)

Reply:
Good spotting I think it might just be the letter G. When I remove the styles and change the network to Sepolia the alignment seems correct.

<img width="306" alt="Screenshot 2023-09-22 at 10 19 29 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/2079e444-d1db-45dc-b7bb-9fa7e417dd91">
<img width="310" alt="Screenshot 2023-09-22 at 10 23 38 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d52ed73f-9273-4db3-8c67-ba6c10c6ec0c">


<img width="177" alt="Screenshot 2023-09-22 at 10 17 19 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/cd8c6981-d043-4438-b4ca-5af0fd08cb24">